apps/chrome-devtools/project.json

```diff
@@ -21,8 +21,8 @@
         "^build",
         "build-panel",
         "build-components",
-        "build-extension",
-        "copy-assets"
+        "copy-assets",
```

Comment:
I hope this change does not have effect.
If you want to apply an order, you should me one depending of the other